CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## v0.7.0 [2022-11-07]
```

Comment:
```suggestion
## v0.7.0 [2022-12-27]
```

Reply:
Overall I think we could skip updating the CHANGELOG file since that can be done as a part of the release process.